.github/actions/build-py/action.yml

```diff
@@ -30,8 +30,16 @@ runs:
           cp dist/*.whl wheelhouse
         fi
         cp -r wheelhouse ..
-    - name: Upload Python Wheels
+    - name: Upload Python Wheels Simulation # See Issue #939
       uses: actions/upload-artifact@v3
+      if: inputs.package == 'simulation'
+      with:
+        name: python-wheels-${{ inputs.package }}
+        path: pycode/wheelhouse
+    - name: Upload Python Wheels
```

Comment:
Cant you name it upload python wheels for both and use an if just for v3 vs v4. this way it would be immediately visible that simulation needs v3...